platform/ios/src/MGLMapView.mm

```diff
@@ -3489,52 +3487,13 @@ - (CLLocationDistance)metersPerPointAtLatitude:(CLLocationDegrees)latitude
     return mbgl::Projection::getMetersPerPixelAtLatitude(latitude, self.zoomLevel);
 }
 
-- (CLLocationDistance)metersPerPixelAtLatitude:(CLLocationDegrees)latitude
-{
-    return [self metersPerPointAtLatitude:latitude];
-}
-
 #pragma mark - Camera Change Reason -
 
 - (void)resetCameraChangeReason
 {
     self.cameraChangeReasonBitmask = MGLCameraChangeReasonNone;
 }
 
-#pragma mark - Styling -
-
-- (NS_ARRAY_OF(NSURL *) *)bundledStyleURLs
-{
-    if ( ! _bundledStyleURLs)
-    {
-        _bundledStyleURLs = [NSMutableArray array];
-        for (NSUInteger i = 0; i < mbgl::util::default_styles::numOrderedStyles; i++)
-        {
-            NSURL *styleURL = [NSURL URLWithString:@(mbgl::util::default_styles::orderedStyles[i].url)];
-            [_bundledStyleURLs addObject:styleURL];
-        }
-    }
-
-    return [NSArray arrayWithArray:_bundledStyleURLs];
-}
-
-- (nullable NSString *)styleID
-{
-    [NSException raise:@"Method unavailable" format:
-     @"%s has been replaced by -[MGLMapView styleURL].",
-     __PRETTY_FUNCTION__];
-    return nil;
-}
-
-- (void)setStyleID:(nullable NSString *)styleID
-{
-    [NSException raise:@"Method unavailable" format:
-     @"%s has been replaced by -[MGLMapView setStyleURL:].\n\n"
-     @"If you previously set this style ID in a storyboard inspectable, select the MGLMapView in Interface Builder and delete the “styleID” entry from the User Defined Runtime Attributes section of the Identity inspector. "
```

Comment:
It would pretty confusing to get a more generic exception when upgrading from v2._x_ to v4.0.0, but hopefully that’s a minuscule number of developers. 😉